tests/subsys/modem/backends/uart/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,38 @@
+/*
+ * Pins P1.10 and P1.11 must be connected to each other to loopback RX/TX.
+ */
+
+/ {
+	aliases {
+		test-uart = &uart1;
+	};
+};
+
+&uart1 {
+	status = "okay";
+	current-speed = <115200>;
+	pinctrl-0 = <&uart1_default>;
+	pinctrl-1 = <&uart1_sleep>;
+	hw-flow-control;
+	pinctrl-names = "default", "sleep";
+};
+
+&pinctrl {
+	uart1_default: uart1_default {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 1, 11)>;
+		};
+		group2 {
+			psels = <NRF_PSEL(UART_RX, 1, 10)>;
+			bias-pull-up;
```

Comment:
Technically you shouldn't need a pull up here...

Reply:
Tested and verified that it is not needed, and have removed it :)